internal/olm/client.go

```diff
@@ -129,7 +130,7 @@ func (c Client) UninstallVersion(ctx context.Context, version string) error {
 	objs := toObjects(resources...)
 
 	status := c.GetObjectsStatus(ctx, objs...)
-	if !status.HasExistingResources() {
```

Comment:
IMHO it should not be change sense as a good practice bool funcs names should be as `Is` or `Has`.
Also, ! hasX is more clear than NoX.